src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1662,21 +1678,23 @@ private void CreateOrUpdateText(string baseId, Vector3 pt, IRenderPackage rp)
                 Model3DDictionary.Add(textId, bbText);
             }
             var geom = bbText.Geometry as BillboardText3D;
+           
             geom.TextInfo.Add(new TextInfo(HelixRenderPackage.CleanTag(rp.Description),
-                pt + defaultLabelOffset));
+               pt + defaultLabelOffset));
         }
 
         private DynamoGeometryModel3D CreateDynamoGeometryModel3D(HelixRenderPackage rp)
         {
-            var meshGeometry3D = new DynamoGeometryModel3D(renderTechnique)
-            {
-                Transform = Model1Transform,
-                Material = WhiteMaterial,
-                IsHitTestVisible = false,
-                RequiresPerVertexColoration = rp.RequiresPerVertexColoration,
-                IsSelected = rp.IsSelected
-            };
-
+          
+                var meshGeometry3D = new DynamoGeometryModel3D(renderTechnique)
+                {
+                    Transform = new MatrixTransform3D(rp.Transform.ToMatrix3D()),
```

Comment:
I think this is safe, since HelixRenderPackage now implements ITransformable and I'm not sure how we would get an old HelixRenderPackage here, but we could write something like this:

```
 Transform = rp is Autodesk.DesignScript.Interfaces.ITransformable ?
                new MatrixTransform3D((rp as Autodesk.DesignScript.Interfaces.ITransformable).Transform.ToMatrix3D()) :
                System.Windows.Media.Media3D.MatrixTransform3D.Identity,
```